lib/seed/Seeder.js

```diff
@@ -76,11 +77,24 @@ class Seeder {
     return name;
   }
 
-  _getStubPath() {
-    return (
-      this.config.stub ||
-      path.join(__dirname, 'stub', this.config.extension + '.stub')
-    );
+  async _getStubPath() {
+    const packageDirectory = await pkgDir(process.cwd());
+    const consumingPackageJson = require(path.resolve(
+      packageDirectory,
+      'package.json'
+    ));
+    const consumingPackageIsModule = consumingPackageJson.type === 'module';
+    const stubExtension = this.config.extension;
+    const stubFormat =
+      stubExtension === 'js'
```

Comment:
can this be broken somehow? ternary inside a ternary is not very easy to read

Reply:
if the extension is js, check to see if the consuming package has type: module in its package.json. if it does, use the module format, if it doesnt, use esm format. Otherwise, just use the stub file with the extension as is.

I think that covers all cases. The problem with breaking it out is that we assign it to a const.
Maybe it's sensible to put it in an iife? either way.